lib/govuk_index/publishing_event_processor.rb

```diff
@@ -2,7 +2,7 @@ module GovukIndex
   class PublishingEventProcessor
     def process(message)
       Services.statsd_client.increment('govuk_index.rabbit-mq-consumed')
-      PublishingEventWorker.perform_async(message.delivery_info[:routing_key], message.payload)
+      PublishingEventWorker.perform_async(message.delivery_info[:routing_key], JSON.parse(message.payload))
```

Comment:
This should NOT be required.

If the message is correctly formatted (i.e. has the JSON_FORMAT content_type) then it has already been parsed from here: https://github.com/alphagov/govuk_message_queue_consumer/blob/master/lib/govuk_message_queue_consumer/json_processor.rb

We should be using the existing functionality if at all possible, as this change could potentially result in trying to parse a message multiple times.